src/mixins/portfolio.js

```diff
@@ -246,15 +248,26 @@ export const createPortfolioMixin = ({
       return Object.keys(ownerMap).sort((a, b) => ownerMap[b] - ownerMap[a]);
     },
     nftCreatorInfoListOfCollected() {
-      return this.nftCreatorAddressListOfCollected.map(id => {
-        const user = this.getUserInfoByAddress(id);
-        return {
-          id,
-          displayName: user?.displayName || id,
-          avatar: user?.avatar || getIdenticonAvatar(id),
-          isCivicLiker: user?.isSubscribedCivicLiker,
-        };
-      });
+      return this.nftCreatorAddressListOfCollected
+        .filter(id => {
+          const user = this.getUserInfoByAddress(id);
+          const likerId = user?.user || id;
+          const displayName = user?.displayName || id;
+          const lowerCaseInputFilter =
+            this.nftCreatorInputFilter?.toLowerCase() || '';
+          return lowerCaseInputFilter
+            ? displayName.toLowerCase().includes(lowerCaseInputFilter) ||
+                likerId.toLowerCase().includes(lowerCaseInputFilter) ||
+                id.includes(lowerCaseInputFilter)
+            : true;
+        })
+        .map(id => {
+          const user = this.getUserInfoByAddress(id);
+          const displayName = user?.displayName || id;
+          const avatar = user?.avatar || getIdenticonAvatar(id);
+          const isCivicLiker = user?.isSubscribedCivicLiker;
+          return { id, displayName, avatar, isCivicLiker };
+        });
```

Comment:
I think you should iterate with map() first, the above code seems redundant as you use most of the properties